ahd2fhir/config.py

```diff
@@ -89,14 +89,35 @@ def get_ssl_context(self):
 
 class Settings(BaseSettings):
     # AHD URL. Should not end with a trailing '/'
-    ahd_url: str = ""
+    ahd_url: str
     # AHD API token
     ahd_api_token: str = ""
     # name of the AHD project
-    ahd_project: str = ""
+    ahd_project: str
+    # description of the project used if
+    # ahd_ensure_project_is_created_and_pipeline_is_started is set to true
+    ahd_project_description: str = "Auto-created by ahd2fhir"
     # name of the pipeline
-    ahd_pipeline: str = ""
+    ahd_pipeline: str
+    # ahd_version
+    ahd_version: str = "5"
```

Comment:
in der docker-compose-dev ist es nun schon version 6

Reply:
in der docker-compose-dev-v6 ist es v6 (weil das für pipeline erzeugung mit der neuen methode nicht mit v5 geht), aber die normale docker-compose-dev ist noch v5 und inhaltlich sollte es damit auch weiterhin funktionieren